apps/ui/src/views/SpaceUser/Statement.vue

```diff
@@ -5,12 +5,17 @@ import { enabledNetworks, getNetwork, offchainNetworks } from '@/networks';
 import { Space, Statement, User } from '@/types';
 import ICAgora from '~icons/c/agora';
 import ICKarmahq from '~icons/c/karmahq';
+import ICSafe from '~icons/c/safe';
 import ICTally from '~icons/c/tally';
 
 const SOURCE_ICONS = {
   agora: { icon: ICAgora, link: 'https://www.agora.xyz' },
   karmahq: { icon: ICKarmahq, link: 'https://karmahq.xyz' },
-  tally: { icon: ICTally, link: 'https://www.tally.xyz' }
+  tally: { icon: ICTally, link: 'https://www.tally.xyz' },
+  safe: {
+    icon: ICSafe,
+    link: 'https://safe-claiming-app-data.safe.global/guardians/guardians.json'
```

Comment:
Not sure what link to use there, https://apps-portal.safe.global/safe-claiming-app/ doesn't load to me, maybe just regular safe app URL?